crates/openvino/tests/util.rs

```diff
@@ -79,3 +82,13 @@ pub const DEFAULT_MARGIN: F32Margin = F32Margin {
 
 /// A helper type for manipulating lists of results.
 pub type Predictions = Vec<Prediction>;
+
+/// OpenVINO's v2024.2 release introduced breaking changes to the C headers, upon which this crate
+/// relies. This function checks if the running OpenVINO version is pre-2024.2.
+pub fn is_version_pre_2024_2() -> bool {
```

Comment:
NIT - Assuming we will need these checks in the future as well for different breakages, having a generic function that takes one or more version arguments might be more beneficial.

Reply:
I don't think we want to improve this: we want to get rid of it. I think the next PR after this one before releasing 0.8.0 should be some kind of solution for dealing with "library-bindings mismatches." In that PR, however it gets solved, let's remove this test-skipping.